introduction.md

```diff
@@ -1,2 +1,5 @@
 This is me. I am me. You are not me. You are you.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?